projects/plugins/jetpack/modules/videopress/editor-media-view.php

```diff
@@ -41,7 +41,7 @@ function videopress_handle_editor_view_js() {
 		array(
 			'home_url_host'     => wp_parse_url( home_url(), PHP_URL_HOST ),
 			'min_content_width' => VIDEOPRESS_MIN_WIDTH,
-			'content_width'     => $content_width,
+			'content_width'     => isset( $content_width ) ? $content_width : 640,
```

Comment:
Since we're editing this, I'm wondering if we should cast this as an int to be a bit more safe?

```suggestion
			'content_width'     => isset( $content_width ) ? (int) $content_width : 640,
```